### PR TITLE
CAKE-5003 Design updates and fixes

### PR DESCRIPTION
**Pull request by mklucsarits**
* Simplified some markup + CSS
 * Implemented changes requested by Design
 * Reverted CMP cookie version

### CODE REVIEW

File: src/i18n/tracking-opt-in.json [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
-        "For additional options, please select %learnMoreButton%. Note that if you do not accept, you may see ads that are less relevant to you and certain features of the site may not work as intended. You can change your mind and revisit your consent choices at any time by visiting our \"%privacyPolicyButton%\"."
+        "For additional options, please select \"%learnMoreButton%.\" Note that if you do not accept, you may see ads that are less relevant to you and certain features of the site may not work as intended. You can change your mind and revisit your consent choices at any time by visiting our %privacyPolicy%. For information about our partners using cookies on our site, please see the %partnerList%."
 ],
-    "acceptButton": "Accept",
+    "acceptButton": "ACCEPT",
     "backButton": "Back",
-    "learnMoreButton": "Learn More",
+    "learnMoreButton": "LEARN MORE",
```
Comment (JoshuaRogan) [inline src/i18n/tracking-opt-in.json]: If we want this in all caps for all languages we might want to do this with `text-transform: uppercase` instead

Comment (mklucsarits) [reply src/i18n/tracking-opt-in.json]: We're already doing that in the CSS for the `<button>`s it's used on, however this string is also used in some paragraph text and we'd like it uppercase there as well, at least for English. I suppose I could dynamically wrap it with a CSS class in that case also.

File: src/components/Preferences.js [hunk 2]
```diff
@@ -11,10 +11,10 @@ function getParagraphs(blockOfText, content) {
             return content[key];
         }
```
Comment (cschucode) [inline src/components/Preferences.js]: Does it make sense to have the same `getParagraphs()` **ScreenOne.js** and **Preferences.js**?

File: src/components/ScreenOne.js [hunk 3]
```diff
@@ -8,13 +8,30 @@ function getParagraphs(blockOfText, content) {
         if (content[key]) {
             return content[key];
         }
```
Comment (cschucode) [inline src/components/ScreenOne.js]: Same logic as **Preferences.js** `getParagraphs()`.

File: src/components/Preferences.js (hunk 2 above)
Comment (mklucsarits) [reply src/components/Preferences.js]: It doesn't make sense I know, but the function uses JSX so we can't move it to a non-JSX utility file (at least I don't think so).

File: src/i18n/tracking-opt-in.json (hunk 1 above)
Comment (cschucode) [reply src/i18n/tracking-opt-in.json]: It might be easier if we stayed consistent in this file with capitalization of buttons.

Comment (mklucsarits) [reply src/i18n/tracking-opt-in.json]: I agree that probably would have been better. We've already resubmitted the translations once and would add more wait time to undo this change.

File: src/components/Preferences.js (hunk 2 above)
Comment (cschucode) [reply src/components/Preferences.js]: Actually, looks like it would be more trouble importing and exporting global styles etc.

**Pull request merged**